### PR TITLE
Moves RCD from autolathe into advanced engineering

### PR DESCRIPTION
**Pull request by Kaffe-work**
## About The Pull Request

This moves the RCD from the hacked autolathe into the Advanced engineering techweb, making it printable at engineering and cargo lathes.  

## Why It's Good For The Game

This removes roundstart access RCD in the hacked autolathe that is available in cargo into the engineering techweb, allowing it to be printed at prolathes instead. It can be printed from both the Engineering and Cargo protolathe, meaning that the same roles has access to them. 

The fact that it is no longer printable from autolathe means fewer greytide will have instant access to RCD, since you could simply unwrench the ORM and hack the autolathe very rapidly. This means that the only ones at roundstart will be the ones in engineering and EVA.  

No changes has been made to techweb costs to compensate. 

## Changelog
:cl: IradT
balance: moved RCD from Autolathe -> Protolathe
/:cl:

### CODE REVIEW

**Comment by Kaffe-work** [comment]
This pr is also intended to come with a upgrade to the RCD that can fit into toolbelts, currently waiting for a sprite

**Comment by Dawson1917** [comment]
I never knew this was a thing thanks to how easy it is to get your hands on these through EVA and engivends. Even removing RCDs from the autolathe, the magic wall-printing and door-deleting devices shouldn't be printable anywhere at all

**Comment by zxaber** [comment]
@Dawson1917 Making it impossible to get extra RCDs will just result in even fewer people bothering to fix bombings and allowing the breach to force a shuttle call. Building 50 walls by hand is tedious as hell.

**Comment by Dawson1917** [comment]
> 
> 
> @Dawson1917 Making it impossible to get extra RCDs will just result in even fewer people bothering to fix bombings and allowing the breach to force a shuttle call. Building 50 walls by hand is tedious as hell.

Cargo exists for this reason, and suffers because there is very little to order that can't be summoned from the ethereal realm by lathes in every department. And besides that, there are 5 or so available to the station at roundstart. The people intended to be carrying one are the people intended to fix bombings.

**Comment by zxaber** [comment]
It's quite often the case that there aren't enough RCDs to go around in Engineering.

Though them being orderable from Cargo sounds like just as good a solution to be honest.

**Comment by Kaffe-work** [comment]
I'm not sure what you think here: do you want less access to RCD? more access? buff cargo? This is imo a step in the right direction, of removing roundstart breakins while still giving engineering decent access to rcd's for when hull breaches start to happen. If there is a hole in medbay, and you have to order them from cargo, that's probably 2 minutes, time for them to arrive + time to convince cargo to actually buy them.

**Comment by Ratvar** [comment]
The issue with RCDs is that door-deleting and place-repairing device is one and the same. Now that there are jaws of life, I think it's safe to remove RCD's ability to make doors disappear while ignoring all whiny powergamers.

**Comment by TheVekter** [comment]
I thought Ratvar was a little silly with that but y'know, the more I think about it, that's a good point. Deconning doors takes some effort without an RCD, but you can just... completely fucking delete them in a few seconds with one. Now that we have other, harder-to-get methods for doing this, and traitors already could with C4, I don't see a need for it existing any longer.

So yeah, +1 on removing RCD airlock deconstruction.

**Comment by Kaffe-work** [comment]
RCD should be used to remove airlocks as well, I disagree. have you never misplaced something in construction, or even more common, given some door the wrong access while fixing shit?

**Comment by Ty-the-Smonk** [comment]
RCDs are the only way to remove station floors to expose a space tile, I use this for xenobio setups

**Comment by Dorsisdwarf** [comment]
I suppose this does solve the issue that every map has to be balanced security wise around anyone having an rcd to instabaleet doors

**Comment by Shadowflame909** [comment]
deconning doors is tedious.

You're never going to stop a mad man with an emag and the ability to bolt doors.

It's unironically one of the strongest hijack methods.

RCD is important because Doors are hard to break through without them.

Also, don't you have to eat up a whole lot of metal and hack the auto-lathe to even get one? 

If someone's putting in the effort to take down a door, and you're not willing to put in the same effort to like. Not leave your very important door unguarded/move any of the items that people would want to steal.

That's your L. Not Mr. Proactives.

**Comment by Kaffe-work** [comment]
on metastation, the most played map, the metal is a non-issue, due to the orm that is nearby.

**Comment by Tlaltecuhtli** [comment]
make manually decon/build door better (same with all that old shit code like apc )(( double the time it would need with rcd )) and give a tool way to decon floor

**Comment by Fhaxaris** [comment]
having to hack an auto-lathe, just to get a tool that is important for your job is really tedious. i still think it should be in both of them though.

**Comment by Cyprex** [comment]
Yeah, I'd rather see more metal reinforcements on airlocks to make RCDing harder(metal shielding on wires which can be quickly added to airlocks stops RCDs until it is removed) as a mapping solution or just removing the ability to delete airlocks with it.

**Comment by Shadowflame909** [comment]
Once more. This PR is 3 things.

An antag buff. Since the bolted door + emag gimmick will become much more stronger without the abillity to just ez delete it.

An incentive for laziness. Oh my god. Someone is literally destroying doors to get guns in the armory. I wish we had a job specifically designed for guarding this one place.

Lastly, it's a non-issue for the grey tide it's trying to address. Because all you've done is locked the door. But the gate right beside it has a big old hole.

RCd's don't need this nerf and I'm quite curious on how hatred of greytide makes this a solution.

Rcd's are slightly faster, if even then just multitooling and crow-baring a door. Since it only requires one item and a lot of metal instead of two.

Literally anywhere you can get into with an RCD. You can do with a multitool.

Even fewer places considering you cannot use an RCD to get into the vault, while you can with a toolbox.

Think of all the malf ai's with their ai cores heavily guarded with bolted doors. and emergency shut-downs.

Think of the mad bomber who specifically went after engineering. Who will stop these cronies?!?

I promise it'll be the guys going for RCD's.

So tldr: Not a greytide nerf since a multitool and a crowbar will literally get you into any place an RCD can just as fast. It's certainly an antagonist buff though.

**Comment by Kaffe-work** [comment]
@shadowflame909

this pr is aimed at moving the tools from the autolathe into the engineering lathe, since engineering should have these tools. 

incidentally, it also means it's harder for people to access RCD. not by much, but a little.

**Comment by Fhaxaris** [comment]
epic valid everyone breaking into engineering / gun-cargo to get the RCD

**Comment by Kaffe-work** [comment]
I assume you valid everyone breaking into science to get Bluespace backpacks as well?

**Comment by Fhaxaris** [comment]
no reason to break into engineering to even get it if it is also in the cargo lathe. everyone will just go break into cargo because it is super easy. no i don't valid people for breaking into a dept unless I think they are an antag.

**Comment by actioninja** [comment]
It being printable from the autolathe in the first place was a mistake.
